packages/cardano-services/src/ChainHistory/DbSyncChainHistory/DbSyncChainHistoryProvider.ts

```diff
@@ -42,7 +46,7 @@ export class DbSyncChainHistoryProvider extends DbSyncProvider() implements Chai
     { paginationPageSizeLimit }: ChainHistoryProviderProps,
     { db, cardanoNode, metadataService, logger }: ChainHistoryProviderDependencies
   ) {
-    super(db, cardanoNode);
+    super({ cardanoNode, db, logger });
     this.#logger = logger;
```

Comment:
The `#logger` property can be removed from from `DbSyncChainHistoryProvider` as it now inherits the `logger` property from `DbSyncProvider`.
The same, can be applied to all other `Provider`s extending `DbSyncProvider`.

Reply:
Good one, applied in 95e990040301ec5bf26de7346cc2b152dae8556e @iccicci